src/bot/commands/roulette.js

```diff
@@ -41,8 +41,8 @@ class Barrel {
 let barrel = new Barrel(barrelSize)
 
 async function resolver({ isBroadcaster, isModerator, username: currentUser }) {
-  if (isBroadcaster || isModerator) {
-    return 'I can\'t kill you my lord <3 ! mindse4Stop'
+  if (isBroadcaster || isModerator || currentUser.toLowerCase() === 'v1dev') {
```

Comment:
Didn't we removed this ?

```suggestion
  if (isBroadcaster || isModerator) {
```

Reply:
Yes but I tried to re-establish my position 🙃